src/test/java/com/google/firebase/FirebaseAppTest.java

```diff
@@ -419,6 +423,96 @@ public void testTokenRefresherStateMachine() {
     assertEquals(0, refresher.cancelCalls);
   }
 
+  @Test(expected = IllegalArgumentException.class)
+  public void testDefaultAppEmptyFile() {
```

Comment:
testEmptyFirebaseConfigFile

Default app means something else in this context. Lets rename these tests in terms of FirebaseConfig -- testEmptyFirebaseConfigString, testEmptyFirebaseConfigObject etc.